src/main/java/com/mycompany/app/service/BasicStockService.java

```diff
@@ -0,0 +1,45 @@
+package com.mycompany.app.service;
+
+import java.util.*;
+import java.text.SimpleDateFormat;
+import com.mycompany.app.model.*;
+
+/**
+ * BasicStockService provides a concrete implementation of StockService interface
+ *
+ * @author Linda M Roseberry
+ */
+
```

Comment:
this class is doing too much work. 
you just needed to do the one getQuote(String symbol) method and it could return dummy date.